.gitignore

```diff
@@ -1,2 +1,15 @@
 /config_*.xml
 /translations/*.php
+
+### PhpStorm ###
```

Comment:
This is not related to the project or the pull request. You can add it in your own `~/.gitignore`

Reply:
I just added the needed bare minimum subset of https://github.com/PrestaShop/PrestaShop/blob/develop/.gitignore. May I open a new PR and add the whole file?

Reply:
I never understood why people add many things in the .gitignore not related to the project.
If you look at Linux repository: https://github.com/torvalds/linux/blob/master/.gitignore
It only contains files related to the project (and compilers like Kdevelop)

Reply:
An example why it's useless. We don't have Jira dependencies :sweat_smile: 
 https://github.com/PrestaShop/PrestaShop/blob/develop/.gitignore#L296

Reply:
So shall I remove it?

Reply:
No keep it for the moment, we must create something in the devdocs about what should be inserted in the gitignore file.